cloudify/workflows/tasks.py

```diff
@@ -32,15 +32,14 @@
     TASK_SENDING,
     TASK_SENT,
     TASK_STARTED,
-    TASK_RESPONSE_SENT,
     TASK_RESCHEDULED,
     TASK_SUCCEEDED,
     TASK_FAILED,
-    TERMINATED_STATES
+    TERMINATED_STATES,
```

Comment:
It's almost like you knew I was going to be reading this...

Reply:
should've made it comma-first